src/OpenLoco/src/Paint/PaintTile.cpp

```diff
@@ -202,18 +202,18 @@ namespace OpenLoco::Paint
             session.finaliseTrackRoadOrdering();
             session.finaliseTrackRoadAdditionsOrdering();
             session.setF003F6(0);
-            if (session.get525CE4(0) != 0xFFFF)
+            auto& bridgeEntry = session.getBridgeEntry();
+            if (!bridgeEntry.isEmpty())
             {
                 if (sub_42AC9C(session))
                 {
                     session.setSegmentSupportHeight(SegmentFlags::all, 0xFFFF, 0);
                 }
-                if (session.getGeneralSupportHeight().height >= session.get525CE4(0))
+                if (session.getGeneralSupportHeight().height >= bridgeEntry.height)
                 {
-                    session.setGeneralSupportHeight(session.get525CE4(0), 0x20);
+                    session.setGeneralSupportHeight(bridgeEntry.height, 0x20);
                 }
-                session.set525CE4(0, 0xFFFF);
-                session.set525CF0(0);
+                session.setBridgeEntry(kNullBridgeEntry);
```

Comment:
Vanilla didn't reset everything here but i don't see why not. The reset isn't used so we might as well just reset it back to 0.